samples/dynamically_add_module_sample/src/main.c

```diff
@@ -11,49 +11,102 @@ int main(int argc, char** argv)
     if (argc != 3)
     {
         printf("usage: dynamically_add_module_sample modulesConfigFile linksConfigFile\n");
-        printf("where configFile is the name of the file that contains the Gateway configuration\n");
+        printf("where modulesConfigFile is the name of the file that contains gateway modules and linksConfigFile is the name of the file that contains gateway links.\n");
+        printf("press ENTER to exit.\n");
```

Comment:
It's not in any of the other samples, is it? And it adds a bunch of noise to the sample. The primary goal of the sample is to show people how to call the new function, not cater to VS developers (they can run it from a command line to see the output, or use Ctrl-F5 instead of F5).